docs/milestone-1.md

```diff
@@ -2,75 +2,78 @@
 
-1. Update environment file `application.properties` with Vault target URL (before starting Scality OSIS application)
-    * Vault URL should be set at property `osis.scality.vault.endpoint` (Refer Sample `application.properties` [here](../src/main/resources/application.properties))
-    * Vault Super admin credentials should be updated at properties `osis.scality.vault.access-key` and `osis.scality.vault.secret-key`
+1. Update the `application.properties` environment file with the Vault target URL (before starting the Scality OSIS application)
+    * Set the Vault URL in the `osis.scality.vault.endpoint` property (see `application.properties` [here](../src/main/resources/application.properties))
+    * Update the Vault super-admin credentials in the `osis.scality.vault.access-key` and `osis.scality.vault.secret-key` properties.
 
 1. Run the Scality OSIS application (Refer [here](../README.md#running-the-application)). Milestone-1 release tag `0.1.0-SNAPSHOT`
 
-1. On OSE machine, vCloud Director OSE and OSE UI should be installed and running successfully.
+1. vCloud Director OSE and OSE UI must be installed and running on the OSE machine.
 
-1. On OSE machine, integrate Scality OSIS with vCloud Director OSE. (Refer [here](#OSIS-integration-with-vCloud-Director-OSE))
+1. Integrate Scality OSIS with vCloud Director OSE on the OSE machine. (Refer [here](#OSIS-integration-with-vCloud-Director-OSE))
 
 ### Validation Steps
 
-1. Login as a vCloud Director Administrator on vCloud Director UI
-1. Navigate to `Resources` tab.
-1. Create a new `Organization` (a.k.a vCloud Director tenant) with required fields.
-1. Navigate to `Object Storage` tab.
-1. `Tenants` pane should be activated and should list all the vCloud Director tenants with associated `Storage Tenant ID` (if exists)
-1. Click on the Tenant name that is created in step 2 under `Tenants` pane.
-1. Enable toggle button beside the Tenant name.
-1. Click `Enable`. (This step will create a Vault Account in the background)
-1. A `Storage Tenant ID` should be generated on the UI which is also the associated Vault Account's Account ID.
-
-### OSIS integration with vCloud Director OSE:
+1. Log in as a vCloud Director Administrator on vCloud Director UI.
+1. Navigate to the `Resources` tab.
+1. Create a new `Organization` (vCloud Director tenant) with required fields.
+1. Navigate to the `Object Storage` tab.
+1. The `Tenants` pane is activated and lists all the vCloud Director tenants with an associated `Storage Tenant ID` (if any exist).
+1. Click the tenant name created in the `Tenants` pane.
+1. Enable the toggle button beside the Tenant name.
+1. Click `Enable`. (This creates a Vault account in the background)
+1. A storage tenant ID, which is also the associated Vault Account's Account ID, is generated on the UI.
```

Comment:
I will still highlight `Storage Tenant ID` as this is a label on UI.
```suggestion
1. `Storage Tenant ID`, which is also the associated Vault Account's Account ID, is generated on the UI.
```

Reply:
Sounds reasonable to be.